docs/source/markdown/podman.1.md

```diff
@@ -64,6 +64,8 @@ Path to ssh identity file. If the identity file has been encrypted, podman promp
 If no identity file is provided and no user is given, podman defaults to the user running the podman command.
 Podman prompts for the login password on the remote server.
 
+The `CONTAINER_SSHKEY` or `DOCKER_CERT_PATH` will override the default. Each is mapped to `CONTAINER_HOST` or `DOCKER_HOST`, respectively.
```

Comment:
I'm not sure what this is saying.  Maybe:

```suggestion
If defined, either the `CONTAINER_SSHKEY` or the `DOCKER_CERT_PATH` environment variables will override the default path for the ssh identity file. Each is mapped to the `CONTAINER_HOST` and the `DOCKER_HOST`, environment variables respectively.
```
I'd add more on how the mapping works, but honestly IDK how that works and I'm not sure the average reader will.  Can you elaborate?